app/assets/images/payment_icons/grabpay.svg

```diff
@@ -0,0 +1 @@
7cHO+VW+Ae+usPAy5dOdnh600z/uZWiGZ+2Wavsthhb F+Hq4pLbzofrVE1N3t3CXyV8pPaUF8vH7FSpTuwXqcK4YOE9XtDQ4DY5OR2dLXATuWhLxPx7kgol 2xlpX7jAZXNDAQq8id2l1lObjK2JX2WZJOzUyiGbj2IgfWoQZFmHkn2NYaotAr2Wj1oayDUXEs5q 6UpILBe17iAJqRujs6xcbOoIYaY363EKKoXyNuOZX4X1cLXQQK+5rVwQaiZrl6tXRgmMI9YI991f e0PL3dWPDlWWTB306wMTGz6d44lejM+0wQUmexIeS+85wRUbLA7HJkmmESI9KpVmTxlzix2Y/trZ Agvxnn/NFcO5ILO16vBTT8cudwi74BseiBcPVodObQ7LHsg1KqR1N1OtPsEticsXlgcQXh53GQuS 0VPGKBSWtRSOTil7jr+oho1zS+jcJyxSbq8JIzdqXb9mbNoFTL8dWCSXR6fB6cKaCsVGZjSyySwr /UiXHr5Lir3NWFytnzNmvdzkueG4KX2ecd3AG6YdwiFzfzTeSaf8WpWFRTKMLsLmtsqvTMVoDF+k LgOg35BFvJzkujTLmTXNBjeF8civ3iXGdpYkcrmEvXCNUtjwhQ+FU7v4dy+9x9jGH9Bv0T98ghbG LFDKGT3p2C0acRB9HoeFMAxddkzSUWRG5l3r0NuMucmo1cB48pZyj7RrJeByKxGrRAY8jNjXCZ06 w78vAe5fsR6lxENvL120cxC+eYF5bq2ghLoD7gSDp4Xw2uycIte7I1cWr/NxBt/qlM0Zt0vUiRiV Qrtm0WKWf8cMlltEHSK4awB3OOE9LqWRrYRBE+jtdA3LULWQGkdd2IavB+bmmP8yydiaOV8ip88t xqGtXCTaiQjXJTa9ViLPDiJTxawE1+rVUOcllADksQI9pXBYipCMblELZtkI5tA3JicerKzLineZ MJPCAJdAuOAjY3cWmcHZrq64aLLQFdrFQ+W0fM+X7/RjgZ51Vg1+YBzjivF1kxzGR09XJr7HKWDz CGqchqzTQSGMw3roEWISBePsiphkbcqtTRTKqvp8gnio8MjmBHHQ+yKDrxg3PODlPdsBJ2B6UAER KzNwgEQPcmvrG9wUXNF3pH3zq/2x/eRgk98zqNani7bZyq/ba6zG8cAYTyhctrqByXFms4eta4I+ VoJzRhgw4NrlQzbIsGs5Jl4wTreSCQnqgkuXrXrx4jg8mYkhMI5DNY63zALGEzKWVfTKGV9fMI6X MrGGDaPe7IrNvsB40HaW5HSLFZ8X4ca1q+GLghD+xbaujlLr1bPA7fuMaV66g5zSrrr6odlXGceK UByL6eGcWTAW2gVRwJV8rUgqdNZMKVKkOM6+zfh0ZCXXCHSKf8d46BQF6K+brThjoV2FRD+S2ZUE 9cyVr32R8QEVIXkVx8vuKrdgpbVKdngex2nGUDWxtuUn8zNl0+1PLLG1Ttd4ilC0QLpWJnHJsM84 xl1287AFVxhDIq4ISxPKWvnYIHCkTcx2lakg14lmOmOhXT0eoAIOUkSw+U/Uqolh3mXsObyGUIVU Zgy/YiKGQkU5qYpQUEFML/Kl14wLiWszRrS+rnXGQrs60Vo2DyrjI4ppoj9N3GcMs+nhKqluLTEG 90l7i9RYjMpDw76rxPpl3c3uMhZB6ogT00fGHr9CEhGcK1qmKuB6jUZ5n/Gyu24c7fBdzcewre0G wutNjhjXfNmhfJWxEOK11n1kLEptvsSKYuJMB2wHP6lV4D5jFIm7FcII4liy9TXGMxraWjjj2RI8 +BgkPSLH1aBdxlyVpMB8ZCxa8eNolVfLhAVPI+ybvMzjPuOeRQMamnkBsYJbRVVfYQx3004yYvXg kmVHfbyfsfircb+drrl2lzE/1pQKpg3GQrsYmo1O6AB3jJChu+D4Ydsl+1rNzgJGRU8cjTFuLkfW 7FzKUw9mHxyl+NllzEVJ28BpjC31iYH+DDxImWMFvFlqi4Jm2mMM+36wvLafMD5jQnLZKYHVS6YM FzZTg1ROfXmNpd/zDcaeTNh+eMEDq1460aRIHQVlRyzffsmY2UjzqT8vRW4Wafl4NYH+QrwlfWbw 2G2GTxe9t6/EccpD0xf5WWbsy0XoRl9oTnMEe6qVblXQK2GyyMQzxnhG5uRQw/ZRUUTB4lQaY3Zm jNVxwJuxs3blCHWX8frwsYWVcz3pZHVrZgD9Rj9IOfWgD/9czN1ctHh05uNpa3Z8xZh1noonIZZL thjjoY9jS81G9Ci11P6FfOyUWSKFksZY0q7tE7wWvWAoFBk/xdh9hUY+Z7wcnZXT2J7PUadp9Uo5 580i2mxkAVlqm9RfqrkE9Q3GknY9eUwfLi2S+HY+9ffzbWzYBJYji/sXjK2L+uRrmzGlk+q2VrrD 7TMmk97J4brFWGhX+ezdKOI92AMQj3hfMaaaJ+/NnzG2/FEZI3s8hfjCToJoT9pivplSGfv899x6 3lWhv95U5msZ/JIxvbk+ZIvX3sgzxnCinrO5KZPuvtHNFxjTAk4K36EVD+JUxmfe4vULM6dxEm/s JXEr+0OIx9an+/35+yckOJ2YEPSvGtNm/bNOjvx07eVbLqSOE1ozpFmOgu2zW9TtG/eEwdrF8XSp 7/3PvxD5s+AbY+XJ5V+NjedPfzfuQnn/tCW/CbXICfX3O/t8+O2av7Lvd/fxOCuv+f3MW8qfBeXE +/D9/j4fMuPqjf/z4XMhMZ5+7n8bfBIE43L+X6ywYJyN7vf7+m8gKock9/rvd2RgYGBgYGBgYGBg YGBgYGBgYGBgYGBgYGBgYGBg8BfjH4/BoK/HnP8JAAAAAElFTkSuQmCC"/><rect x="0" y="0" width="38" height="24" rx="2" ry="2" fill="none" stroke="#000" stroke-width="1" stroke-opacity=".7"/></svg>
```

Comment:
- The border doesn't look the same as other icon. See this [video demonstration](https://screenshot.click/20-48-ivxf5-zwa2m.mp4). You can add this code in your SVG. For more info, see the [contribution](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template) document.

```svg
<path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
<path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>
```

- The payment icon looks big. Is it possible to resize?

[![alt](https://screenshot.click/20-53-7bzsw-qgxaa.png)](https://screenshot.click/20-53-7bzsw-qgxaa.png)